pymagicc/api.py

```diff
@@ -1,18 +1,16 @@
-import platform
 import shutil
 import subprocess
 from distutils import dir_util
 from os import listdir, makedirs
-from os.path import join, exists
+from os.path import basename, dirname, exists, join
 from tempfile import mkdtemp
 
 import f90nml
 import pandas as pd
 
-from .compat import get_param
-from .paths import _magiccbinary, _magiccpath
+from .config import config
 
-_WINDOWS = platform.system() == "Windows"
+IS_WINDOWS = config['is_windows']
```

Comment:
Why is "Windows" a config option?

Reply:
We could call it "IS_WINDOWS" everywhere, btw :-)

Reply:
A check for windows was being performed in several different modules so it was moved to config. In my eyes, knowing what OS is being used is still configuration even if it won't change during the lifetime of the program